apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -472,6 +472,13 @@ parent_generation() ->
     Fun = fun() -> get_consensus_config_key([<<"parent_chain">>, <<"parent_generation">>]) end,
     aeu_ets_cache:get(?ETS_CACHE_TABLE, parent_generation, Fun).
 
+acceptable_sync_offset() ->
+    Fun = fun() -> get_consensus_config_key([<<"parent_chain">>, <<"acceptable_sync_offset">>], 60000) end,
+    aeu_ets_cache:get(?ETS_CACHE_TABLE, acceptable_sync_offset, Fun).
+
+child_epoch_length(Length) ->
```

Comment:
Should this be saved "on chain"?
If one later synchronizes with an existing HC, then one probably needs to know what child epoch length was at a given height? If so, we may want this information stored on chain.

A possible place to store this is having it in the election contract.
Another possibility, but more intrusive, is to have a special state tree for this.

Reply:
I agree it makes sense to have the sync offset and the epoch length on chain.